### PR TITLE
Addition of Error Code

### PR DESCRIPTION
**Pull request by joinimran**
Add information of error code. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9189

### CODE REVIEW

File: windows/deployment/update/windows-update-errors.md [hunk 1]
```diff
@@ -37,4 +37,4 @@ The following table provides information about common errors you might run into
...
 |                0x80244007                |   WU_E_PT_SOAPCLIENT_SOAPFAULT    | SOAP client failed because there was a SOAP fault for reasons of WU_E_PT_SOAP_\* error codes. |                                                                                        This issue occurs because Windows cannot renew the cookies for Windows Update.  <br><br>Review [KB2883975](https://support.microsoft.com/help/2883975/0x80244007-error-when-windows-tries-to-scan-for-updates-on-a-wsus-serv) for instructions to resolve the issue.                                                                                         |
-
+|                0x80070422               |       | This issue occurs when the Windows Update service stops working or not running. |                                                                                       Check if Windows Update service is running.<br>                                                                                        |
```
Comment (JohanFreelancer9) [inline windows/deployment/update/windows-update-errors.md:40]:
```suggestion
|                0x80070422               |       | This issue occurs when the Windows Update service stops working or is not running. |                                                                                       Check if the Windows Update service is running.<br>                                                                                        |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by jaimeo** [comment]
@joinimran does this added error code not have an accompanying message?

**Comment by illfated** [comment]
This error code has been around since 2015, but even so, there is no extra error message information provided.

See also: https://answers.microsoft.com/en-us/windows/forum/windows_10-update-winpc/windows-10-does-not-install-updates-error/80fcf8c8-21d7-4e22-bceb-1dd88255658b

> There were some problems installing updates, but we'll try again later. If you keep seeing this and want to search the web or contact support for information, this may help: (0x80070422)

**Pull request merged**